xml/tuning-ceph-tuning.xml

```diff
@@ -265,17 +265,34 @@ min_alloc_size_hdd=65536
         </para>
       </warning>
    </sect2>
-   <sect2>
+   <sect2 xml:id="bluestore-cache">
      <title>BlueStore Cache</title>
      <para>
-       With many workloads, increasing the OSD cache can improve
-       performance. This cache is a read cache that is populated
-       on object read by default. The memory target is inclusive of
-       the base RAM required for the OSD process and the total cache.
-       By setting the cache minimum higher than the default, it is
-       guaranteed that the value specified will be the minimum cache
-       available for each OSD. The idea being that more low-probability
-       cache hits may occur.
+       Increasing the &bluestore; cache size can improve performance with
+       many workloads. While &filestore; OSD's cache data in the kernel's page
+       cache, &bluestore; OSDs cache data within the memory allocated by the
+       OSD daemon itself. The OSD daemon will allocate memory up to its
+       memory target (as controlled by the <literal>osd_memory_target</literal> parameter),
+       and this determines the potential size of the &bluestore; cache.
+       The &bluestore; cache is a read cache that, by default, is populated
+       by object read. By setting the cache minimum higher than the default,
```

Comment:
"object read" sounds weird to me. 

"when objects are read"?

I also suggest "setting the cache's minimum size".

Reply:
`... object's read`?

Reply:
I vote for: "It is populated when objects are read" as the non-engineer equivalent of "It is populated by object read" (which is engineer-speak).